src/sundials/sundials_context.c

```diff
@@ -134,35 +169,22 @@ int SUNContext_SetProfiler(SUNContext sunctx, SUNProfiler profiler)
   sunctx->own_profiler = SUNFALSE;
 #endif
 
-  return (0);
+  return SUN_SUCCESS;
 }
 
-int SUNContext_GetLogger(SUNContext sunctx, SUNLogger* logger)
+SUNErrCode SUNContext_GetLogger(SUNContext sunctx, SUNLogger* logger)
 {
-  if (sunctx == NULL)
-  {
-    return (-1);
-  }
```

Comment:
We should probably keep the checks for a non-null `SUNContext` in these `SUNContext_` functions, but in other places (as I mentioned somewhere else) there may be no point in checking if the context is NULL since we cannot really do anything without one.